style.css

```diff
@@ -97,3 +112,178 @@ li:hover {
 li:active {
   color: #2230d2;
 }
+
+/* work section */
+.work {
+  background-color: #e5e5e5;
```

Comment:
- [ ] please use the correct work section background color

Reply:
changed in to the main color on figma #f7f7f9;